src/components/item_page/ItemPage.scss

```diff
@@ -0,0 +1,59 @@
+@import 'styles/_shared.scss';
```

Comment:
I don't have a preference, but just out of curiosity, is it more idiomatic to use styled React components vs external CSS?

Reply:
I think it's more about personal preference than what's considered idiomatic, but I personally like using styled components.